app/logical/sources/alternates/twitter.rb

```diff
@@ -0,0 +1,67 @@
+module Sources
+  module Alternates
+    class Twitter < Base
+      TWITFIX_DOMAINS = %w[fxtwitter.com ayytwitter.com twitter64.com vxtwitter.com pxtwitter.com twittpr.com nitter.net]
+      NITTER_HOSTS = %w["nitter.net nitter.moomoo.me nitter.kavin.rocks nitter.it nitter.domain.glass nitter.ca nitter.fdn.fr"]
+
+      def force_https?
+        true
+      end
+
+      def _nitter_domains
```

Comment:
I thought I doubl posted the comment and deleted, but apparently I didn't.

There's no need for the underscore to be here.

Reply:
I figured it would be best as a private method, and thought that's how those were denoted. I'll rename

Reply:
Ruby does have private methods. https://github.com/zwagoth/e621ng/blob/d86112d1d3f119d23b2a4f66294c1727b9b1df8d/app/inputs/search_form_builder.rb

Everything after the private keyword is, well, private. Something to keep in mind is that this only properly works for instance methods. It does not work like this for class methods.

Reply:
Oh yeah, of course. I forgot that. It's in the pixiv helper too, my bad, was thinking in python this morning